assignments/boolean-birds/code.js

```diff
@@ -16,3 +16,25 @@
  *   - Globby Birds eat fish, mice, and nuts.
  *
  */
+const isFlobbyBird =(isRed, isSpotted) => isRed && isSpotted
```

Comment:
```suggestion
const isFlobbyBird = (isRed, isSpotted) => isRed && isSpotted;
```
That shows some very small changes I'd make to that line of code. I'll talk about how to read these soon. But basically I'd add a space after the `=` and add a `;` at the end.